scripts/zkai_attention_kv_fused_softmax_table_route_matrix_gate.py

```
@@ -0,0 +1,821 @@
+            "matched_source_plus_sidecar_raw_proof_bytes_total": sum(
+                row["source_plus_sidecar_raw_proof_bytes"] for row in matched_rows
+            ),
+            "matched_fused_proof_size_bytes_total": sum(row["fused_proof_size_bytes"] for row in matched_rows),
+        },
+        "validation_commands": list(VALIDATION_COMMANDS),
+    }
+    validate_core(result)
+    return result
+
+
+def mutation_cases(result: dict[str, Any]) -> tuple[tuple[str, Any], ...]:
```

Ruff (0.15.12)

[warning] Unused function argument: `result`

(ARG001)